cli/js/read_text_file.ts

```diff
@@ -7,14 +7,12 @@ export function readTextFileSync(path: string | URL): string {
   const file = openSync(path);
   const contents = readAllSync(file);
   file.close();
-  const decoder = new TextDecoder();
-  return decoder.decode(contents);
+  return new TextDecoder().decode(contents);
```

Comment:
In fact, the module should declare a `const decoder = new TextDecoder();` and be used here and the next function instead of creating a new decoder each call.